pkg/setup/flags.go

```diff
@@ -52,6 +54,12 @@ func (f *Flags) Register(flags *pflag.FlagSet) {
 		"",
 		"Specify a custom name for the cluster.",
 	)
+	flags.DurationVar(
+		&f.timeout,
+		"timeout",
+		5 * time.Second,
+		"Specify HTTP timeout (default 5s).",
```

Comment:
Probably should say something like, `"Specify HTTP timeout in seconds (default 5 seconds)." Think that would be a little clearer that this option is in seconds

Reply:
It's not limited to seconds only it could be any go duration string